redis/src/main/scala/zio/redis/api/Strings.scala

```diff
@@ -16,13 +16,12 @@
 
 package zio.redis.api
 
-import zio.duration._
 import zio.redis.Input._
 import zio.redis.Output._
 import zio.redis.ResultBuilder._
 import zio.redis._
 import zio.schema.Schema
-import zio.{Chunk, Has, ZIO}
+import zio.{Chunk, ZIO, _}
```

Comment:
```suggestion
import zio._
```

@jxnu-liguobin this pattern repeats a lot, please adjust it everywhere according to the suggestion.